molter/help.py

```diff
@@ -0,0 +1,197 @@
+import functools
+import logging
+
+import dis_snek
+from dis_snek import Embed
+from dis_snek.ext.paginators import Paginator
+
+import molter
+
+__all__ = ("HelpCommand",)
+
+log = logging.getLogger(dis_snek.const.logger_name)
+
+
+class HelpCommand:
+    show_hidden: bool
+    """Should hidden commands be shown?"""
+    show_disabled: bool
+    """Should disabled commands be shown?"""
+    run_checks: bool
+    """Should commands be checked if they can be run by the help command user?"""
+    show_self: bool
+    """Should this command be shown in the help message?"""
+    show_params: bool
+    """Should parameters for commands be shown?"""
+    show_aliases: bool
+    """Should aliases for commands be shown?"""
+    show_prefix: bool
+    """Should the prefix be shown?"""
+
+    embed_title: str
+    """The title to use in the embed. {username} will be replaced by the client's username."""
+    not_found_message: str
+    """The message to send when a command is not found. {cmd_name} will be replaced by the requested command."""
+
+    _client: dis_snek.Snake
+
+    def __init__(
+        self,
+        client: dis_snek.Snake,
+        *,
+        show_hidden: bool = False,
+        run_checks: bool = False,
+        show_self: bool = False,
+        show_params: bool = False,
+        show_aliases: bool = False,
+        show_prefix: bool = False,
+        embed_title: str | None = None,
+        not_found_message: str | None = None,
+    ) -> None:
+        self._client = client
+        self.show_hidden = show_hidden
+        self.run_checks = run_checks
+        self.show_self = show_self
+        self.show_params = show_params
+        self.show_aliases = show_aliases
+        self.show_prefix = show_prefix
+        self.embed_title = embed_title or "{username} Help Command"
+        self.not_found_message = not_found_message or "Sorry! No command called `{cmd_name}` was found."
```

Comment:
Minor nitpick: this phrasing is *slightly* unprofessional, but... yeah, that's a nitpick.

Reply:
I mean, 2 objections to that.
One, exclamation marks are *not* unprofessional. Theyre frequently used in throughout professional settings in order to reduce end-user stress when something doesnt work. I believe you mean its not emotively neutral, which also leads into the second point. 

Two, Snek as a whole is *not* professional, so the point is quite void.